m3.yml

```diff
@@ -14,6 +14,8 @@
 #   type:  type of thing does the profile describe
 #   version:  version of the profile
 
+schema_version: "1.0.beta"
```

Comment:
what's the difference between this and the `version:` on [L15](https://github.com/samvera-labs/houndstooth/pull/17/files#diff-23771d2cb9e5c4238105ed88f20a8e9bL15)?

Reply:
This is the schema/specification version, the other is for the specific profile, eg. UCSBs profile.

Reply:
this is the version of M3? got it.

could we clarify that some way? i think "schema" is ambiguous.

Reply:
m3_version? Plus a better description?

Reply:
I think that's a great idea.  m3_version is definitely less confusing.